ci/common/suite.sh

```diff
@@ -11,14 +11,35 @@ FAIL_SUMMARY=""
 END_MARKER="$BUILD_DIR/.tests_finished"
 FAIL_SUMMARY_FILE="$BUILD_DIR/.test_errors"
 
+ANSI_CLEAR="\033[0K"
+
+travis_fold() {
+  local action="$1"
+  local name="$2"
+  name="$(echo -n "$name" | tr '\n\0' '--' | sed 's/[^A-Za-z0-9]\{1,\}/-/g')"
```

Comment:
@jamessan It must not be `printf "$name"`, should be `printf '%s' "$name"` if you aim at random text. But I know for sure that `$name` always starts with `/` in our code, so no need to bother.